gnomad_qc/v3/annotations/generate_freq_data.py

```diff
@@ -42,22 +44,75 @@
 
 def make_faf_index_dict(faf_meta: List[Dict[str, str]]) -> Dict[str, int]:
     """
-    Create a look-up Dictionary for entries contained in the filter allele frequency annotation array
+    Creates a look-up Dictionary for entries contained in the filter allele frequency annotation array
 
     :param List of Dict faf_meta: Global annotation containing the set of groupings for each element of the faf array (e.g., [{'group': 'adj'}, {'group': 'adj', 'pop': 'nfe'}])
     :return: Dictionary of faf annotation population groupings, where values are the corresponding 0-based indices for the
         groupings in the faf_meta array
     :rtype: Dict of str: int
     """
 
-    index_dict = index_globals(faf_meta, dict(group=["adj"]))
-    index_dict.update(index_globals(faf_meta, dict(group=["adj"], pop=POPS)))
-    index_dict.update(index_globals(faf_meta, dict(group=["adj"], sex=SEXES)))
-    index_dict.update(index_globals(faf_meta, dict(group=["adj"], pop=POPS, sex=SEXES)))
+    index_dict = {
+        **index_globals(faf_meta, dict(group=["adj"])),
+        **index_globals(faf_meta, dict(group=["adj"], pop=POPS)),
+        **index_globals(faf_meta, dict(group=["adj"], sex=SEXES)),
+        **index_globals(faf_meta, dict(group=["adj"], pop=POPS, sex=SEXES)),
+    }
 
     return index_dict
 
 
+def make_freq_index_dict(freq_meta: List[Dict[str, str]]) -> Dict[str, int]:
```

Comment:
This is a good point. We actually do, to make sure the `freq_index_dict` global is complete so that the `set_female_metrics_to_na()` function works on all sex-grouped annotations. What I'll do is move the `make_freq_index_dict()` function to `gnomad_methods` and import it in both the release script and here.